### ISSUE TITLE
Destination based on calendar event

### ISSUE THREAD

**Issue by yummypasta** [mrdis/MMM-google-route#5]
Originally suggested in PR #3, but I moved it here to an issue because no code from the comparison branch is necessary.

This issue is under development on the [**notifications**](https://github.com/mrdis/MMM-google-route/tree/notifications) branch.

**Comment by mrdis** [comment]
@yummypasta I tested using a google calendar and fixed a few issues. 
Now the route to the destination specified in the location field of the event is shown as soon as the page is loaded.
I removed the code that tried to parse the "geo" field because I am not able to test it or find any specification for it.
Does it work for you?

**Comment by yummypasta** [comment]
Hi @mrdis,
Did it work for you? I tested it and it didn't seem to change anything.
Since `Log.log` doesn't work, you can "log" by overriding `info.innerHTML`. When I did that, I could see that the [`notificationRecieved`](https://github.com/mrdis/MMM-google-route/blob/8252bc0d5a3328660690f314aff6a0164c2e95bb/MMM-google-route.js#L207) function isn't getting called, and override is `null`.

**Comment by mrdis** [comment]
Yes, it worked for me, but actually I forgot to mention that I used a different, more generic, configuration variable than the one you proposed.
In case you didn't, you should add 
`listen:["CALENDAR_EVENTS"]`
to the module configuration.

Regarding the logs, I connect to the magic mirror server using my browser and watch the logs in the developer console. It's the fastest way for me.
